README.md

```diff
@@ -12,39 +12,44 @@ This website is meant to give people not familiar with our department an idea of
 The target audience that motivated its creation and potential hires. While the site can be used to cater to other needs
 as well, the hiring one should always be kept in mind.
 
-### Who can edit and who is reposible
+### Who can edit and who is responsible
 
 You can treat the website as if it where a wiki. In fact, the previous version of the site was on a wiki. Use your own
-best judgement. If you arr fixing a typo, just push to master. If you think it warrants a second eye or you are activly
+best judgement. If you arr fixing a typo, just push to master. If you think it warrants a second eye or you are actively
 looking for feedback, create a pull request. Everyone part of the WMDE GitHub organization can merge pull requests.
 
 The main authors or the existing content are @samu-wmde and @JeroenDeDauw, so these are the primary suspects for feedback.
 
 Changes to this git repo do not automatically get deployed. You can just leave your changes as they are, to be deployed
 whenever the next deployment happens, or you can pester either @addshore or @JeroenDeDauw to do a deployment.
 
-### Running the website
+### Running the application
 
 Running the site requires PHP 7.0 or later.
 
 After cloning the repo, run `composer install` in the root directory.
 
-For running the application, execute `php -S 0:8000 -t web` in the root directory. This makes the site avilable at `http://localhost:8000/` in your browser.
+For running the application, execute `php -S 0:8000 -t web` in the root directory. This makes the site
+available at `http://localhost:8000/` in your browser. For development, you probably want to use the
+development entry point `url:8000/index.dev.php`, which enables debug mode and disables HTTP caching.
```

Comment:
Well it depends on how you are running it. I'm using Docker (since I don't have PHP installed locally) which makes the full thing `http://172.17.0.2:8000/index.dev.php`